controller/space_areas_test.go

```diff
@@ -154,14 +74,47 @@ func searchInAreaSlice(searchKey uuid.UUID, areaList *app.AreaList) *app.Area {
 	return nil
 }
 
-func createSpaceArea(name string, desc *string) *app.CreateSpaceAreasPayload {
+func (rest *TestSpaceAreaREST) TestListAreas() {
+	t := rest.T()
+	resource.Require(t, resource.Database)
 
-	return &app.CreateSpaceAreasPayload{
-		Data: &app.Area{
-			Type: area.APIStringTypeAreas,
-			Attributes: &app.AreaAttributes{
-				Name: &name,
-			},
-		},
+	/*
+		Space X --> Area 2 ---> Area 21-0
+	*/
+	var createdAreaUuids []uuid.UUID
+
+	parentArea := createSpaceAndArea(t, rest.db)
+	createdAreaUuids = append(createdAreaUuids, parentArea.ID)
+
+	parentID := parentArea.ID
+	name := "Area 2000A"
```

Comment:
Thanks @hectorj2f 
There are restrictions on uniqueness, the reason I used hardcoded names is because it helps me explain the structure in comments way I did here :)
https://github.com/almighty/almighty-core/pull/913/commits/492b99b9986051d453bb0ff38accbf5db18d7043#diff-f3af47db51708ed00b7371ac273318cfR80